### DISCUSSION TITLE
posts/best-download-manager-for-linux/

### DISCUSSION THREAD

**Discussion by giscus** [tazihad/tazihad.github.io#5] [Announcements]
# posts/best-download-manager-for-linux/

For years I have searched for best download manager on linux. I think I have found it. It has multithreading feature and many more. Yes, I am talking about aria2. Let’s setup aria2 and integrate it with linux. We are going to use podman and aria2 in container. It takes very little resources, 650 KB memory when idle. Let’s set it up.

https://zihad.com.bd/posts/best-download-manager-for-linux/

**Comment by RandomlyHuman** [comment]
Hi, I'm a newbie in Linux, but I had the problem that I had to run the last two lines to run the container every time I turned on the computer. 

What I did was (I know this is a crude solution, but it's a workaround) to create a shell script and put the two lines in it

------------------------------------------------
#!/bin/bash

   # Wait a little while to make sure systemd is ready
   sleep 10

   # Reload the systemd configuration
   systemctl --user daemon-reload 

   # Start the aria2-pro service
   systemctl --user start aria2-pro.service
---------------------------------------------------


Then make the executable
chmod +x ~/scripts/start_aria2.sh

And ready, start the Zihad script. 

## Who by the way I thank a lot for sharing.
